[DEV-230] Update FAQ

Texian opened the pull request:
[TICKET](https://artistguide.atlassian.net/browse/DEV-230)

**Description:**

Update FAQ page with new layout, questions, and answers

**Screenshots (if applicable):**

Screenshots here

**PR Checklist:**

- [√ ] If you haven't made changes to `package.json`, `package-lock.json` is not present in this PR
- [√ ] You have tested these changes locally by running the app and letting it build successfully
- [√ ] Eslint ran successfully upon commit and `--force` was not used to force this PR

_Make sure you **squash and merge** rather than simply merging all commits at once_.

At least **1** PR approval is required by a Tech Lead to be merged. **2** approvals total are required.

bestfoodalex commented on .github/workflows/codeql-analysis.yml:
```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```
Make sure to uncommit this file

bestfoodalex reviewed (commented):
Looks good other than the codeql analysis file - just uncommit it and LGTM 👍🏻

Texian commented:
Ok, I've removed the unnecessary file and its directory and updated my .gitignore with that particular file name should it resurface somehow. I'm using an app on my iPad to do the updates/merges, so I'm a little unsure if it did something wonky, but if it worked like I expected it to, the change should be ready to merge and the ticket can finally be closed.
